src/NuGet.Core/NuGet.DependencyResolver.Core/Remote/RemoteDependencyWalker.cs

```diff
@@ -155,17 +174,25 @@ public Task<GraphNode<RemoteResolveResult>> WalkAsync(LibraryRange library, NuGe
                             runtimeName,
                             runtimeGraph,
                             ChainPredicate(predicate, node, dependency),
-                            innerEdge));
+                            innerEdge,
+                            transitiveCentralPackageVersions));
                     }
                     else
                     {
+                        // In case of conflict because of a centrally managed version
```

Comment:
Can you elaborate here? 
Is there a test capturing this scenario that I can debug into?

Reply:
Few tests:
RestoreCommanTests
RestoreCommand_DowngradeIsNotErrorWhen_DowngradedByCentralDirectDependency
RestoreCommand_DowngradeIsErrorWhen_DowngradedByCentralTransitiveDependency

RemoteDependencyWalkerTests
WalkAsyncAddsTransitiveCentralDependency